arrows/vxl/image_io.cxx

```diff
@@ -138,6 +143,111 @@ convert_image_helper(const vil_image_view<bool>& src,
   dest = src;
 }
 
+std::string
+plane_filename(const std::string filename, unsigned p)
```

Comment:
pedantic: this function seems unique among the new code in not using spaces inside `()`s... and doesn't even omit them consistently; would be nice if the new code followed the style guide